internal/infrastructure/datastore/db.go

```diff
@@ -33,18 +33,19 @@ func getUrl(config *config.Config) string {
 	)
 }
 
+// func NewDB(cfg *config.Config) (db *gorm.DB) {
 func NewDB(cfg *config.Config) (db *gorm.DB) {
 	var err error
 	dbUrl := getUrl(cfg)
 
 	if db, err = gorm.Open(pg.Open(dbUrl),
 		&gorm.Config{
-			Logger:                 logger.NewGormLog(cfg),
+			Logger:                 logger.NewGormLog(true),
 			SkipDefaultTransaction: true,
 			// CreateBatchSize:        50,
 			TranslateError: true,
 		}); err != nil {
-		log.Error().Msgf("Error creating database connector: %s", err.Error())
+		slog.Error("Error creating database connector", err.Error())
```

Comment:
This does not work as expected. I'm getting `!BADKEY` with this example code:

```
package main

import (
	"errors"

	"golang.org/x/exp/slog"
)

func main() {
	err := errors.New("Example")
	slog.Error("Error creating database connector", err.Error())
}
```

```
$ bin/testing 
2023/11/14 10:42:55 ERROR Error creating database connector !BADKEY=Example
```

Reply:
Fixed

Reply:
I have checked the last change and it worked; resolving this comment; but if further change were necessary I encourage to open a new ticket to address it.